source/includes/_pis.md

```diff
@@ -55,6 +55,26 @@ We support account identification using `UK.OBIE.SortCodeAccountNumber`. We don'
 
 You can only make payments in `GBP`. We don't support other currencies.
 
+### Domestic Payments Limits
+
+There are default limits for daily outbound payments:
+
+| Account type               | Default limit |
+| ---------------------------| ------------- |
+| Personal                   | £10000        |
+| Joint                      | £10000        |
+| Business (sole trader)     | £25000        |
+| Business (limited company) | £50000        |
```

Comment:
looking at the payment limits it seems like `Sole Trader` is £25k, `Private Limited Company` is 50k and all other businesses are 10k

https://github.com/monzo/wearedev/blob/0ca069b21ff81240fe1562858fca5e2f22a4c753/service.payment-limit/limits/faster_payments.go#L35-L41

Reply:
Hm, maybe API then should expose more company types, not just `Business` and `Sole Trader...

Reply:
Only "Sole Trader" and "Private Limited Company" are supported at the moment (https://monzo.com/i/business/eligibility/), so only those will be shown in the table.